.github/workflows/sanity.yml

```diff
@@ -112,4 +118,16 @@ jobs:
         stat ./Metabase.dmg
         date | tee timestamp
     - name: Calculate SHA256 checksum
-      run: sha256sum ./Metabase.dmg | tee SHA256.sum
\ No newline at end of file
+      run: sha256sum ./Metabase.dmg | tee SHA256.sum
+    - run: sudo apt install -y dmg2img
+    - run: dmg2img Metabase.dmg Metabase.img
```

Comment:
took me hours to discover that modern DMG can't be consumed by 7-zip directly, as it needs to be converted first to "a more sensible" image.